### DISCUSSION TITLE
Inquiry about Compatibility with Konyks eCosy Device

### DISCUSSION THREAD

**Discussion by matguig** [arendst/Tasmota#20114] [Support request]
Hello,

I'm reaching out to the Tasmota community for the first time and would like to introduce myself as a new member interested in exploring and contributing to the Tasmota ecosystem.

I have recently been looking through the list of compatible devices for Tasmota on the project's page but couldn't find any information regarding the compatibility with the Konyks eCosy device (more information available here: https://konyks.com/produit/ecosy/). This device operates on the TYWE3S module.

I am curious to know if there are any known issues or potential complications that I should be aware of before attempting to integrate this device with Tasmota. Any guidance or information regarding the compatibility, or what challenges I might face, would be greatly appreciated.

Thank you for your time and assistance.

Best regards,
Matthieu

**Comment by sfromis** [comment]
TYWE3S is compatible for flashing Tasmota, but after that it needs to be configured to match the hardware. Looks like it is Tuya-based, which often means a separate MCU as the main chip controlling the device, where Tasmota does support working with this. Still may need some setup to match the functions available, and only doing it could reveal if there are hurdles.

**Comment by barbudor** [comment]
Is that a french brand ? 🐓 
Fun fact: I was looking details on that device which I found on the Australian version of Amazon with a lot of photos in french (packages, key features, ...) 😆 

Nice because I wasn't aware of such decvice compliant with the Pilote Wire (I believe it's something quite Frenchy that is not used elsewhere).
The basic modes of the Pilote Wire could be done with a 2-relay board, like a Shelly Plus 2PM, and 2 diodes but would still missing the nice touch buttons on the front panel.
And a Tasmotized Shelly Plus would support bluetooth so you could directly pair a Mija BLE T&H sensor, with a few rules you can control your heater.

**Reply by barbudor** [reply]
@matguig I believe that you have already openned it
Could you kindly share photos of the teardown ?
